### ISSUE TITLE
Bonus Chapter - Implementing a Health Check API

### ISSUE THREAD

**Issue by BobMolloy** [markjprice/cs11dotnet7#8]
Page 2, Bullet 7

Every time I run this, I receive the output "Unhealthy" and a 503 status code.

**Comment by markjprice** [comment]
Are you able to call the actual endpoint and get a response?

**Comment by BobMolloy** [comment]
I’m not entirely sure that I understand the question. Here is what I see:

**Comment by markjprice** [comment]
There is nothing shown. Please try replying in GitHub and add the screenshot.

**Comment by BobMolloy** [comment]
![hdyf](https://user-images.githubusercontent.com/7458504/209978097-7df42eb8-79b2-417e-b16f-9d36d7f722ed.png)
![hdyf2](https://user-images.githubusercontent.com/7458504/209978114-76be8705-46c7-474f-b9d8-ba2a780c7e65.png)
![hdyf3](https://user-images.githubusercontent.com/7458504/209978132-930c54d1-75fd-4a0c-9acf-d2eb6e6a6125.png)

**Comment by markjprice** [comment]
Okay, so just before the **Implementing advanced features for web services** section that is online only was the client HTTP section. I assume you were therefore able to call the the actual endpoint and get a response, for example, the HTTP client could call: GET https://localhost:5002/api/customers/?country=UK and the response was returned. Therefore the endpoint is working. If you then added the package for health checks and the Web API service is returning 503 Unhealthy then it's is "lying" in some way. After getting 503, try changing the address box to: https://localhost:5002/api/customers/?country=UK and see what response you get and please screenshot that.

**Comment by BobMolloy** [comment]
![uk1](https://user-images.githubusercontent.com/7458504/209980365-fcfd99a9-5751-4962-a22e-94e92d73f54f.png)


![uk2](https://user-images.githubusercontent.com/7458504/209980384-428061d2-cd91-4bbd-801c-e7fd354969d5.png)


![uk3](https://user-images.githubusercontent.com/7458504/209980411-3f0a0819-33e8-4974-9f08-ed16af00bc58.png)

**Comment by markjprice** [comment]
Yes, so the endpoint is actually healthy but the Microsoft health check thinks that it not. I don't know why. I'll see if I can research why that might be.
